synapse/_scripts/synapse_port_db.py

```diff
@@ -779,6 +783,11 @@ def alter_table(txn: LoggingTransaction) -> None:
             await self._setup_sequence("receipts_sequence", ("receipts_linearized",))
             await self._setup_sequence("presence_stream_sequence", ("presence_stream",))
             await self._setup_auth_chain_sequence()
+            await self._setup_sequence(
+                "application_services_txn_id_seq",
+                ("application_services_txns",),
+                "txn_id",
```

Comment:
guessing this this argument is optional and defaults to `stream_id`?

Reply:
oh, it's in literally the next diff hunk :facepalm:

Reply:
I did debate about using a keyword argument for it.